src/converters/cover.ts

```diff
@@ -173,6 +199,23 @@ class CoverHandler implements ServiceHandler {
   updateState(state: Record<string, unknown>): void {
     this.monitors.forEach((m) => m.callback(state));
 
+    if ('motor_state' in state) {
+      const latestMotorState = state['motor_state'] as string;
+      switch (latestMotorState) {
+        case 'opening':
+          this.motorState = CoverHandler.MOTOR_STATE_OPENING;
+          break;
+        case 'closing':
+          this.motorState = CoverHandler.MOTOR_STATE_CLOSING;
+          break;
+        case 'stopped':
+        case 'pause':
+        default:
+          this.motorState = CoverHandler.MOTOR_STATE_STOPPED;
+          break;
+      }
+    }
```

Comment:
I don't fully get the point of this `switch` case, as the static readonly variables you've added represent the same values, right?
For the cases, you should probably use those as well, and then you can probably simplify it a bit as well.

Something like...

```suggestion
      switch (latestMotorState) {
        case CoverHandler.MOTOR_STATE_OPENING:
        case CoverHandler.MOTOR_STATE_CLOSING:
          this.motorState = latestMotorState;
          break;
        default:
          this.motorState = CoverHandler.MOTOR_STATE_STOPPED;
          break;
      }
    }
```